eml/api/api.js

```diff
@@ -207,3 +207,80 @@ export const ifSubscribed = async (userId, courseId) => {
     }
   }
 };
+
+//CREATED BY VIDEOSTREAM TEAM
+/**
+ *
+ * @param {* name of video in bucketRoute, should be the same as lecture id} fileId
+ * @param {* use 360p, 720p or 1080p - standard is 360p} resolution
+ */
+export const getVideoDownloadUrl = (fileName, resolution) => {
+  let usableResolution = '360x640';
+
+  switch (resolution) {
+  case '180p':
+    usableResolution = '180x320';
+    break;
+  case '360p':
+    usableResolution = '360x640';
+    break;
+  case '720p':
+    usableResolution = '720x1280';
+    break;
+  case '1080p':
+    usableResolution = '1080x1920';
+    break;
+  default:
+    usableResolution = '360x640';
+    break;
+  }
+  const _vidUrl = `${url}/api/bucket/stream/${fileName}_transcoded${usableResolution}.mp4`;
+  console.log(_vidUrl);
+  return _vidUrl;
+};
+
+//CREATED BY VIDEO STREAMING TEAM
+export const getSectionAndLecturesBySectionId = async (sectionId) => {
```

Comment:
Im writing the comment here, but this also relates to SectionCard, LectureSwipeScreen and the backend. Instead of fetching section and lectures from the DB this way, you can just fetch the lectures using the section ID, you don't need the section, since you don't actually use anything from the section object. And if you need to use anything from the section object, you can parse it from SectionCard to LectureSwipeScreen instead of fetching it from the DB (again). Also, in the backend instead of casting to an object, the model should be updated so you don't have to do the casting. 
Please comment if this should be an Issue instead of a comment in here :)

Reply:
@Donhusum  Please make it an issue, I agree and it will be fixed at a later point

Reply:
You can probably just make it an issue, since it won't break anything

Reply:
I have created an issue with this :)